docs/user-guide/styling-your-docs.md

```diff
@@ -153,6 +153,52 @@ Your directory structure should now look like this:
     `theme`. If you want to remove files, or build a theme from scratch, then
     you should review the documentation for building a [custom theme].
 
+#### Overriding Template Blocks
```

Comment:
It would be nice to reference this section from the `custom-themes.md` file. I would like it if we could encourage external themes to use the same block names. This may happen anyway, but would be good to call out.